src/name_bazaar/ui/events.cljs

```diff
@@ -192,10 +209,32 @@
  interceptors
  (fn [{:keys [db]}]
    (let [addr (get-in db [:active-page :route-params :user/address])]
-     (info ["Try address" addr (web3/address? addr)])
+     (info ["TRY ADDR RESOLUTION" addr (web3/address? addr)])
      {:db db
       :dispatch
       (if-not (web3/address? addr)
         (let [addr-patched (tldize addr)]
           [:ens.records/load [(namehash addr-patched)] {:load-resolver? true}])
-        [:public-resolver.record.addr/loaded])})))
+        [:public-resolver.addr.record/load addr])})))
+
+(reg-event-fx
+ :try-resolving-my-address
+ interceptors
+ (fn [{:keys [db]}]
+   (let [addr (get-in db [:active-address])]
+     (info ["Trying my address" addr (web3/address? addr)])
+     {:db db
+      :dispatch
+      (if (web3/address? addr)
+        [:public-resolver.addr.record/load addr]
+        [:public-resolver/nodata])})))
+
+(reg-event-fx
+ :start-routing
```

Comment:
There another issue opened for this https://github.com/district0x/name-bazaar/issues/91. But ok can stay here meanwhile